tasks/configuration.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: create groups
+  group:
+    name: '{{ item.name }}'
+    system: '{{ item.system | default(omit) }}'
+    gid: '{{ item.gid | default(omit) }}'
+    state: '{{ item.state | default("present") }}'
+  tags:
```

Comment:
We don't add  tags on task level, only in the main.yml.

Reply:
removed